server/json/user1-institutions.json

```diff
@@ -6,47 +6,53 @@
       "status": {
         "code": 0,
         "message": ""
-      }
+      },
+      "currentSubmission": 0
```

Comment:
Sorry, I might be missing something else here. But the status is already there right? In the 
```
"status": {
  "code": 0,
  "message": ""
}
```

Anyway, I'll merge this and we can dig more with the backend crew when the API doc is ready.